js/bitmart.js

```diff
@@ -500,75 +499,44 @@ module.exports = class bitmart extends Exchange {
     }
 
     async fetchContractMarkets (params = {}) {
-        const response = await this.publicContractGetContracts (params);
```

Comment:
Sorry, I used the wrong endpoint, the old endpoint doesn't exist anymore, so I used the tickers endpoint instead because it looks like the closest they have

Reply:
It actually does exist, but it fails from time to time (happened before). I'll get that escalated to the Bitmart team.

Reply:
Nope, nevermind. It is disabled permanently. So, we should clean it out.